components/products.jsx

```diff
@@ -98,34 +97,35 @@ export const ProductCard = ({ shopifyProduct, rechargeProduct }) => {
     actions: { addToCart, openCart, toggleCart }
   } = useCart();
 
-  const [recurringText, setRecurringText] = useState(
-    rechargeProduct
-      ? `every ${pluralizeText(
-          rechargeProduct.subscription_defaults.charge_interval_frequency,
-          rechargeProduct.subscription_defaults.order_interval_unit,
-          `${rechargeProduct.subscription_defaults.order_interval_unit}s`
-        )}`
-      : null
+  const recurringText = rechargeProduct
+    ? `every ${pluralizeText(
+        rechargeProduct.subscription_defaults.charge_interval_frequency,
+        rechargeProduct.subscription_defaults.order_interval_unit,
+        `${rechargeProduct.subscription_defaults.order_interval_unit}s`
+      )}`
+    : null;
+  const { title, description, images: imageEdgesContainer } = shopifyProduct;
+  //This takes the max variant price if no price is found. Not perfect
+  //but okay for the purposes of a demo for now.
+  const getPriceString = 'variants.edges[0].node.price';
+  const shopifyPrice = lodashGet(
+    shopifyProduct,
+    getPriceString,
+    shopifyProduct?.priceRangeV2?.maxVariantPrice?.amount ?? 0
   );
```

Comment:
Something like this would make more sense:

```
  const shopifyPrice = get(shopifyProduct, 'variants.edges[0].node.price') ?? get(shopifyProduct, 'priceRangeV2.maxVariantPrice.amount') ?? 0;
```

`get` your long path, use `get` for each of the long paths you access here to make them safe, fall back with the nullish coalescing operators.